src/messages/Gallery/Gallery.tsx

```diff
@@ -37,7 +38,7 @@ const Gallery: FC = () => {
 		}
 	}, [carouselContentId, config?.settings.enableAutoFocus]);
 
-	if (elements.length === 0) return null;
+	if (!elements || elements.length === 0) return null;
```

Comment:
according to the interface, it can be undefined or array. So the safety check is not needed there.
I will add anyway, just if we will something in the future.